tsdb/query_executor.go

```diff
@@ -210,14 +209,15 @@ func (q *QueryExecutor) executeSelectStatement(statementID int, stmt *influxql.S
 	}
 
 	// Plan statement execution.
-	p := influxql.NewPlanner(q)
-	e, err := p.Plan(stmt, chunkSize)
+	p := NewPlanner(q.store)
```

Comment:
New Planner swapped in here, which means the system is using the refactored query engine.